packages/blazor-workspace/README.md

```diff
@@ -1,161 +1,7 @@
-<div align="center">
-    <p><b>ni | nimble | blazor</b></p>
-</div>
-
-# Nimble and Spright Blazor
-
-[![Nimble Nuget Version](https://img.shields.io/nuget/v/NimbleBlazor.svg)](https://www.nuget.org/packages/NimbleBlazor)
-[![Spright Nuget Version](https://img.shields.io/nuget/v/SprightBlazor.svg)](https://www.nuget.org/packages/SprightBlazor)
-
-NI-styled UI components for Blazor applications
+# Blazor Workspace
 
 This repo contains:
-1. Blazor components and styles matching the NI brand. These are published as a Nuget package to be consumed by either Blazor WebAssembly or Blazor Server applications.
+1. Blazor components and styles matching the NI brand. These are published as Nuget packages to be consumed by either Blazor WebAssembly or Blazor Server applications.
```

Comment:
I'd vote to at least mention the names of the nuget packages. Optionally we could describe their purposes. For example:
```suggestion
1. Blazor components and styles matching the NI brand. These are published as Nuget packages to be consumed by either Blazor WebAssembly or Blazor Server applications.
    - `NimbleBlazor` contains general-purpose components
    - `SprightBlazor` contains special-purpose or application-specific components
```